pkg/archive/archive_windows.go

```diff
@@ -23,6 +24,17 @@ func CanonicalTarNameForPath(p string) (string, error) {
 	return strings.Replace(p, "\\", "/", -1), nil
```

Comment:
that's probably out of scope of this PR. but you're right. I'll submit a PR separately.